RGPosadas/LitterRally
# TASK-63: Messages Main Menu Frontend

This PR closes #225.

**Steps to test**

1. Start the application
2. Click the messages icon on the navbar
3. You should see both lists for DMs and event chats
4. They should be swipeable and do not change the route
5. Clicking on the events should bring you to the event chats
6. Make sure that group chats and announcements work as they did before
7. Join other events to see if you could chat in another event

**Notes**
The main menu is implemented using swipeable views instead of routes due to problems we were having with nested routes.
